Assets/Scripts/Audio/AudioLoader.cs

```diff
@@ -6,38 +6,40 @@
 using UnityEngine;
 using System.Collections;
 
-public class AudioLoader {
+public class AudioLoader 
+{
+
 	const string DIRECTORY = "Audio/";
```

Comment:
In other files, you shorten "DIRECTORY" TO "DIR" -- Better to stick with one and be consistent.